src/main/java/edu/harvard/pallmall/seeders/DatabaseSeeder.java

```diff
@@ -0,0 +1,51 @@
+package edu.harvard.h2ms;
+
+import edu.harvard.h2ms.domain.core.Method;
+import edu.harvard.h2ms.repository.MethodRepository;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import static java.util.Arrays.asList;
+
+
+/**
+ * Seeds initial data into the database, example taken from
+ * https://dzone.com/articles/how-to-create-a-database-seeder-in-spring-boot
+ */
+@Component
+public class DatabaseSeeder {
+    private MethodRepository methodRepository;
+
+    @Autowired
+    public DatabaseSeeder(
+            MethodRepository methodRepository
+            ) {
+        this.methodRepository = methodRepository;
+            }
+
+    @EventListener
+    public void seed(ContextRefreshedEvent event) {
+        seedMethodsTable();
+    }
+
+    private void seedMethodsTable() {
+        List<List<String>> records = asList(
+                asList("Soap and water", "Handwashing method that involves soap and water in a sink."),
+                asList("Alcohol", "Hand sanitizing station")
```

Comment:
Now that I see this do we really need a name and a description? Any thoughts @2018swecapstone/everyone?